packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -0,0 +1,25 @@
+module KLDivergence = {
+  let logFn = Js.Math.log
+  let subtraction = (a, b) => Ok(a -. b)
+  let multiply = (a: float, b: float): result<float, Operation.Error.t> => Ok(a *. b)
+  let logScoreDirect = (a: float, b: float): result<float, Operation.Error.t> =>
+    if a == 0.0 {
+      Error(Operation.NegativeInfinityError)
+    } else if b == 0.0 {
+      Ok(b)
```

Comment:
This is an approximation, there should be a comment here. 
It probably would be good, just for readability, to return ``Ok(0.0)`` instead of ``Ok(b)``, even though they are the same thing.